### ISSUE TITLE
Difference in accuracy between tensorflow DeepLabv3+ and this version

### ISSUE THREAD

**Issue by ahanagemini** [kazuto1011/deeplab-pytorch#62]
Do you know why there is a difference in accuracy between your network and the tensorflow implementation of DeepLabv3+.

**Comment by kazuto1011** [comment]
I just list the possible factors in terms of the model.
* What is the version of the backbone FCN?
    * ResNet v1, ResNet v2 (Pre-act), etc.
    * ResNet v1 has some variants. My v3+ implementation is based on DeepLab v2 built with Caffe. Its ResNet has 7x7 conv while the official one does not. The stride position is also different.
* Pre-trained on ImageNet or/and COCO?
* What is the output stride in training/testing?

**Comment by ChunmengLiu1** [comment]
> Do you know why there is a difference in accuracy between your network and the tensorflow implementation of DeepLabv3+.

Hi, I'm implementing the coco pre-trained deeplabv3+. However, I got very strange results, about 7% mIoU. Would you like to provide some information for me if you could?  Thank you for your help.

**Comment by kazuto1011** [comment]
* Are the COCO-trained parameters successfully loaded into the backbone part (ResNet)? The `strict=False` option in loading weights hides name inconsistency.
* DeepLab v3 and v3+ have new batch norm params inside ASPP, which are not pre-trained on COCO. Did you add them to the optimizer's param list as trainable weights?
